app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
```

Comment:
Place each argument on the new line.

 def __init__(self,
Comment
Place each argument on the new line.

```
def __init__(
      self,
      comfort_class: int | float,
      clean_mark: int | float,
      brand: str
) -> None:
```